src/util/string_helper.h

```diff
@@ -161,6 +161,15 @@ class StringHelper {
     static char ToUpper(char c) {
         return std::toupper(c);
     }
+
+    /**
+     * Merges two strings together, separated by a comma, if both aren't empty. Otherwise, returns the non-empty string, or an empty string if both are empty.
```

Comment:
```suggestion
     * Merges two strings together, separated by a new line, if both aren't empty. Otherwise, returns the non-empty string, or an empty string if both are empty.
```